app/Validation/SupportEmailValidation.php

```diff
@@ -21,8 +21,7 @@ class SupportEmailValidation
         ],
         "support-phone" => [
             "required" => "El teléfono es obligatorio",
-            "numeric" => "El campo teléfono solo debe contener carácteres numéricos",
-            "exact_length" => "El teléfono debe tener 10 dígitos"
+            "regex_match" => "El campo teléfono debe contener 10 dígitos numéricos y puede incluir una lada opcional"
```

Comment:
```suggestion
            "regex_match" => "El campo teléfono debe contener 10 dígitos y puede incluir una lada opcional"
```
Todo muy bien, solo podrías cambiar esta parte ya que es algo reduntante, pues un dígito ya es numérico